core/src/main/java/org/frankframework/receivers/Receiver.java

```diff
@@ -1825,41 +1817,6 @@ protected synchronized DistributionSummary getProcessStatistics(int threadsProce
 		return processStatistics.get(threadsProcessing);
 	}
 
-	protected synchronized DistributionSummary getIdleStatistics(int threadsProcessing) {
-		DistributionSummary result;
-		try {
-			result = idleStatistics.get(threadsProcessing);
-		} catch (IndexOutOfBoundsException e) {
-			result = null;
-		}
-
-		if (result==null) {
-			while (idleStatistics.size()<threadsProcessing+1) {
-				int threadNumber = idleStatistics.size()+1;
-				result = configurationMetrics.createThreadBasedDistributionSummary(this, FrankMeterType.RECEIVER_IDLE, threadNumber);
```

Comment:
Misschien voor de volledigheid ook de `FrankMeterType.RECEIVER_IDLE` verwijderen?

Reply:
Dan heb ik de vraag: waarom worden ENUM constants TOTAL_MESSAGE_IN_ERROR, TOTAL_MESSAGES_PROCESSED en TOTAL_MESSAGES_REJECTED nergens gebruikt? Kunnen die misschien ook weg?

Reply:
Ja die zijn weggevallen nu alles in de andere meters zit 👍🏻